GoMeet/src/main/webapp/create-meeting-step1.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <!-- Create Meeting Step 1: Set meeting name and add guests -->
+  <head>
+    <title>GoMeet Create Meeting Step 1</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="js/create-meeting-step1.js"></script>
+  </head>
+  <body onload="addGuest(document);">
+    <h1 style="text-align:center;">GoMeet</h1>
+    <h2>Create Meeting</h2>
+    <h3>Step 1: Set meeting name and add guests</h3>
+    <form id="meeting-name-form">
+      <label for="meeting-name">Enter meeting name: </label>
+      <input type="text" name="meeting-name" id='meeting-name'>
+    </form>
+    <br> 
+    <form id="meeting-host-form">
+      <label for="meeting-host">Enter your email: </label>
+      <input type="text" name="meeting-host" id='meeting-host'>
```

Comment:
You can consider using an `type="email"` input here - that input type will do simple checks to verify that the input was in the format of an email (i.e. contains '@', no spaces, no special characters, etc).

Reply:
Done :)